src/components/List/List.tsx

```diff
@@ -20,26 +21,63 @@ export default function List(props: React.PropsWithChildren<ListProps>) {
 
   const baseClass = "list";
 
-  let listTag;
+  let listTag,
+    errorText = "";
```

Comment:
Nitpick: each variable should be declared on it's own line without the comma separator.

Reply:
Why?  Both are valid.  Our linter didn't indicate this as being an issue.  Is this some kind of best practice set up for the Design System specifically?

Reply:
Declaring each variable on its own line is the syntax we've been following. I know the linter doesn't complain but just for consistency's sake.

Reply:
OK, I updated it.